features/daily/backup.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Backup
+
+    Scenario: Log Rotate Backup Files
+        Given there are backup files
+        Then backup files successfully rotated
+
+    Scenario: Backup and restore database
+        Given the asset file "xivo-backup-manager" is copied on the server into "/tmp"
+        When I execute database backup command
+
+        Given there are permanent authentication users with info
```

Comment:
Why using permanent user? The user is only cleanup after the scenario, not after the step

Reply:
restoring the backup deletes the created user and it fails on the cleanup (user not found) after the scenario. Hence why I created the permanent user step.

Reply:
If the cleanup fail, the scenario doen't fail
The error is just an output to the stdout (the default action when cleanup fail). I see 2 other possibilities
* do nothing and keep the error in the log
* check if in previous step (specific to backup) we can set `context.on_cleanup_error` to hide error. But we must ensure that change only affect the current scenario.

I'm just not a fan to create a custom step to avoid cleanup to only avoid stdout message

Reply:
Check if in previous step (specific to backup) we can set `context.fail_on_cleanup_errors=False` to hide error. But we must ensure that change only affect the current scenario. So maybe we need to switch the value after scenario.
Other possibility is to create a tag and change the `fail_on_cleanup_errors` value before and after the scenario with the tag

I don't know if the test should be fail or not if a cleanup fail ...